api/src/routes/__tests__/wish.test.js

```diff
@@ -15,36 +16,53 @@ describe('Wish route', () => {
       illness: 'crecent',
       age: '12'
     },
-    type: wishType,
+    type: firstWishType,
     details: 'i want to be a real star',
     sponsor: {
       name: 'krabs',
       logo: 'K',
       links: []
+    },
+    createdAt: '2019-06-14T18:08:56.374Z',
+    updatedAt: '2019-06-14T18:08:56.374Z',    
     }
-  }
+  const secondWishType = 'see'
   const secondWish = {
     child: {
-      firstName: 'patrick sdfasdf',
-      lastName: 'star',
-      hometown: 'marietta',
-      illness: 'crecent',
-      age: '12'
+      firstName: 'spongebob',
+      lastName: 'squarepants',
+      hometown: 'atlantic ocean',
+      illness: 'non-absorbent',
+      age: '12',
     },
-    type: wishType,
+    type: secondWishType,
     details: 'i am a goofy goober',
     sponsor: {
       name: 'krabs',
       logo: 'K',
       links: []
-    }
+    },
+    "createdAt": "2018-07-10T18:08:56.374Z",
+    "updatedAt": "2018-07-10T18:08:56.374Z",
+  }
+  const thirdWishType = 'meet'
+  const thirdWish = {
+    type: thirdWishType,
+    createdAt: '2010-06-14T18:08:56.374Z',
+    updatedAt: '2019-05-14T18:08:56.374Z',    
+  }
+  const fourthWishType = 'be'
+  const fourthWish = {
+    type: fourthWishType,
+    createdAt: '2017-08-14T18:08:56.374Z',
+    updatedAt: '2018-08-14T18:08:56.374Z',    
   }
 
   beforeEach(async () => {
     await Wish.deleteMany({})
   })
 
-  test('It should respond with an array of wishes', async () => {
+  xtest('It should respond with an array of wishes', async () => {
```

Comment:
did you mean to leave the `x` on `xtest`?

Reply:
x means skip I think

Reply:
not really.  i had to disable that test until the update-resource pull request got merged.

I'll update this branch